cmd/cli/plugin/package/main.go

```diff
@@ -10,12 +10,12 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
 	cliv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/cli/v1alpha1"
+	capdiscovery "github.com/vmware-tanzu/tanzu-framework/capabilities/discovery"
```

Comment:
I think `capdiscovery` is all right. Wanted to throw in an idea, just in case you also like it better: `capdisc`.

Reply:
lets keep the alias as `capdiscovery`